articles/azure/azs-how-deploy-asdk.md

```diff
@@ -20,28 +20,26 @@ The Azure Stack Development Kit (ASDK) is a single server instance of Azure Stac
 
 The ASDK is used in two modes within UKCloud:
 
-* Physical hardware (pre-production) - Community support with Microsoft
+- Physical hardware (pre-production) - Community support with Microsoft
 
-* Virtual servers (development and testing) - not officially supported by Microsoft
-This document covers two scenarios, building from scratch and redeploying.
+- Virtual servers (development and testing) - not officially supported by Microsoft This document covers two scenarios, building from scratch and redeploying.
 
-## Step 1 - Hardware pre-reqs
+## Step 1 - Hardware pre-requisites
 
-Detailed specs are here - https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-deploy
+Detailed specifications are here - [https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-deploy](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-deploy)
 
-|Device  |Details  |
-|---------|---------|
-|CPU    |  >=16 cores       |
-|RAM     | >=128GB        |
-|OS Disks |  >=200GB       |
-|Data Disks  | >=4 * >=250G        |
+| Device     | Details         |
+|------------|-----------------|
+| CPU        | \>=16 cores     |
+| RAM        | \>=128GB        |
+| OS Disks   | \>=200GB        |
+| Data Disks | \>=4 \* \>=250G |
 
-**Virtualisation Note:** VMware HW must be >=11, CPU virtual extension pass-through must be enabled
+**Virtualisation Note:** VMware HW must be \>=11, CPU virtual extension pass-through must be enabled
 
-> [!IMPORTANT]
-> If you deploy your VM with Hardware Version 11 and Operating System Family as Windows Server 2016 you will not be able to create S2D Cluster as disk UUIDs are not exposed.
+> \[!IMPORTANT\] If you deploy your VM with Hardware Version 11 and Operating System Family as Windows Server 2016 you will not be able to create S2D Cluster as disk UUIDs are not exposed.
```

Comment:
Why did you add the \? We don't normally need to escape these characters

Reply:
My bad, I forgot how terrible "format document" in VSCode is for Markdown..... Fixed now